### PR TITLE
EFR: changes to proto were made to fix the issues pointed out by Andrii

### PR DESCRIPTION
**Pull request by pequetreques**
go.work was removed and just a simple **go.mod** was added.   [requested by @Merteg; thanks to @JuanC1303 for his advice]
gen/proto directory was removed in favour of just **proto**.   [requested by @Merteg]
health-service.proto was renamed as just **health.proto**.   [requested by @Merteg]
HealthServiceApi was renamed as just **HealthService**.   [requested by @Merteg]
**Register** and **Push** endpoints were added, and the rest were deleted (sorry for have not read the user story first).   [requested by @Merteg]
**google.protobuf.BoolValue** was used instead of plain bool (I did not know how to deal with it yesterday, so I changed it to bool on purpose just to make @JuanC1303 happy).   [requested by @Merteg]

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module pl-health-service/gen/proto
+module workspace
```
Comment (Merteg) [inline go.mod]: let's name it as repo github.com/Merteg/pl-health-service

File: go.mod [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 
 go 1.19
```
Comment (Merteg) [inline go.mod]: you can do 1.20 now

File: proto/health.proto [hunk 3]
```diff
@@ -1,20 +1,30 @@
...
     string empty = 1;
 }
 
-message EmptyResponse {
+message PushRequest {
+    repeated Health health = 1;
+}
+
+message PushResponse {
     string empty = 1;
```
Comment (Merteg) [inline proto/health.proto]: you can have a message with no fields so no need to add this empty param

**Review by Merteg — CHANGES_REQUESTED** [review]

File: proto/health.proto (hunk 3 above)
Comment (pequetreques) [reply proto/health.proto]: Thanks, I did not know that and also wondered if that was possible.

File: proto/health.proto [hunk 4]
```diff
@@ -1,29 +1,38 @@
...
 
-message EmptyRequest {
-    string empty = 1;
+message RegisterRequest {
+    repeated Target target = 1;
 }
 
-message EmptyResponse {
+message RegisterResponse {
     string empty = 1;
```
Comment (Merteg) [inline proto/health.proto]: same here...

File: proto/health.pb.go [hunk 5]
```diff
@@ -0,0 +1,786 @@
...
+	}
+	return nil
+}
+
+type PushResponse struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	Empty string `protobuf:"bytes,1,opt,name=empty,proto3" json:"empty,omitempty"`
```
Comment (Merteg) [inline proto/health.pb.go]: You didn't regenerate the proto...

**Review by Merteg — CHANGES_REQUESTED** [review]

File: proto/health.pb.go (hunk 5 above)
Comment (pequetreques) [reply proto/health.pb.go]: Sorry

**Review by pequetreques — COMMENTED** [review]
Done

**Review by Merteg — APPROVED** [review]

**Review by JuanC1303 — APPROVED** [review]

**Pull request merged**